### ISSUE TITLE
Using hifiasm for triploid phasing assembling, --n-hap 3 doesn't work

### ISSUE THREAD

**Issue by liuxin-21d** [chhylp123/hifiasm#321]
Hi,
Thanks for your nice hifiasm.
I was using hifiasm to assemble triploid recently, and i used --n-hap 3, 
but i just got *hap1* and *hap2* without *hap3*.
It seems that hifiasm works not very well with triploid phasing assembling?
Or should i configure other special parameters for triploid phasing assembling?
Looking forward to your reply.
Best regards,
Liu Xin

**Comment by chhylp123** [comment]
No, hifiasm cannot output 3 haplotypes right now in default. If you would like to get phased assemblies for triploid samples, you need additional phasing information like the genetic map. Here is a nice example: https://github.com/baozg/Potato_C88.

**Comment by liuxin-21d** [comment]
> genetic

Thank you for your reply and suggestion.

**Comment by liuxin-21d** [comment]
i will try it then. Thank you for your reply and suggestion.
